### DISCUSSION TITLE
stack.where triggers computation?

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [gjoseph92/stackstac#39] [Q&A] [answered]
![computewhereCapture](https://user-images.githubusercontent.com/72196131/116186097-1cfe6600-a762-11eb-994a-4d5915abab8c.PNG)

Does someone know why?

the NDVIMIN is this 
``` python
stack["NDVIMIN"] = stack["NDVI"].min(dim="time", skipna=True)
```

**Comment by gjoseph92** [comment, marked as answer]
I'm guessing because you said `drop=True`. In order to know which elements to drop, xarray has to first compute them.

**Comment by RichardScottOZ** [comment]
ah, good point!  Thanks! :)
